uwblueprint/supportive-housing
# Employee Directory - Sort By Last Modified

## Notion task link


[Employee Default Sort By Last Modified](https://www.notion.so/uwblueprintexecs/Employee-Default-Sort-By-Last-Modified-f2289846799b40589a9d69d9033d1089)



## Implementation description
* Currently, the employee directory is sorted in random order. With these changes it should sort by last modification date, with the newest / most recently edited logs at the top.



## Steps to test
1. Add employee, they should append to the top
2. Edit employee, they should move to the top
3. Activate / Deactivate employee, they should move to the top



## What should reviewers focus on?
* Now that editing a log brings it to the first page, shouldnt the user also be brought to the first page?


## Checklist
